README.md

```diff
@@ -170,7 +170,7 @@ To run tests
 git clone https://github.com/HashNuke/heroku-buildpack-elixir
 export BUILDPACK="$(pwd)/heroku-buildpack-elixir"
 git clone https://github.com/jesseshieh/heroku-buildpack-testrunner
-git clone https://github.com/kward/shunit2
+git clone https://github.com/jesseshieh/shunit2
```

Comment:
what's the reason for moving to your repo here?

Reply:
The original version uses /bin/sh which doesn't handle things like arrays and `declare` etc. All I did was change it to bash. See https://github.com/jesseshieh/shunit2/commit/61f1a1441ffaaf6af8f0a8f6f237c77843dfe50b